cmd/bundle-extractor/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/configmap"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
```

Comment:
now that the `opm` command has merged, should this be `opm bundle extract`?

Reply:
Updated!

Reply:
This has been re-updated to be `opm alpha bundle extract`.